### DISCUSSION TITLE
C# is the vast majority of my Repo but its not listed as a Language

### DISCUSSION THREAD

**Discussion by DavuKnight** [github-linguist/linguist#6043] [Classification] [answered]
https://github.com/DavuKnight/mod_AzerothServiceBus 

Is predominantly a C# Project there are a couple C files mixed in but they are definetely in the minority However my project is classified as a C/C++ project without any reference to C#. The whole point of this project is to allow >DotNet Developers access to this code the Language Identified on the project is very important in my context given its one of the few C# game dev environments on GitHub and as such am hoping to get some momentum. 

How do I get linguist to correctly Identify me *.cs files to be C#

**Comment by Nixinova** [comment, marked as answer]
The C# files would be ignored because they are in the `deps/` folder. See https://github.com/github/linguist/blob/master/docs/overrides.md#vendored-code for how to unhide them in a repo's lang stats.
